test/e2e/upgrade/common.go

```diff
@@ -242,12 +243,26 @@ func checkServicePorts(t *testing.T, name string) {
 	assert.Equal(t, constants.InfinispanAdminPort, int(adminPort[0].Port))
 }
 
-func checkBatch(t *testing.T, name string) {
+func checkBatch(t *testing.T, infinispan *ispnv1.Infinispan) {
+	configMapName := infinispan.Name + "-cm"
+	configMap := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      configMapName,
+			Namespace: infinispan.Namespace,
+		},
+		Data: map[string]string{
+			batchCtrl.BatchFilename: "create cache --file=/etc/batch/mycache.xml mycache",
+			"mycache.xml":           "<infinispan><cache-container><distributed-cache name=\"mycache\" mode=\"SYNC\" /></cache-container></infinispan>",
```

Comment:
I think we should be able to simplify this to:

`"mycache.xml": "<distributed-cache />"`

Reply:
good idea! thanks